Configure

```diff
@@ -2109,7 +2109,7 @@ if ($builder eq "unified") {
         my $index_re = qr/\[\s*(?P<INDEX>(?:\\.|.)*?)\s*\]/;
         my $cond_re = qr/\[\s*(?P<COND>(?:\\.|.)*?)\s*\]/;
         my $attribs_re = qr/(?:\{\s*(?P<ATTRIBS>(?:\\.|.)*?)\s*\})?/;
-        my $value_re = qr/\s*(?P<VALUE>.*?)\s*/;
+        my $value_re = qr/(?P<VALUE>.*?)/;
```

Comment:
Why are the spaces being dropped in this pattern?

Reply:
Because it's the only of these patterns that have this, which makes for harder to fathom patterns further down.  So without this change of `$value_re`, you can do this:

``` perl
            qr/^\s* SUBDIRS \s* = ${value_re} $/x
```

With the change in `$value_re`, we force this:

``` perl
            qr/^\s* SUBDIRS \s* = \s* ${value_re} \s* $/x
```

I would say that the latter, while being a tiny bit more verbose, is also clearer.

Reply:
That makes sense.